### PR TITLE
KupreichikA hw5 ex2

### PR DESCRIPTION
**Pull request by akupreychik**

### CODE REVIEW

File: homework/Kupreichik Anton/HomeWork_5/ex2_Book/service/MenuService.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    static String name, author;
+
+    static public void setNumbersOfBooks(){
+        System.out.println("Какое кол-тво книг вы желаете ввести?");
+        numberOfBooks = in.nextInt();
+        book = new Book[numberOfBooks + 1];
+    }
+
+    public static void enterData(){
+        for (int i = 0; i < numberOfBooks; i += 1) {
```
Comment (TLittleProgrammer) [inline homework/Kupreichik Anton/HomeWork_5/ex2_Book/service/MenuService.java]: i += 1?

File: homework/Kupreichik Anton/HomeWork_5/ex2_Book/bean/Book.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package com.company.bean;
+
+import java.util.Objects;
+
+public class Book {
```
Comment (TLittleProgrammer) [inline homework/Kupreichik Anton/HomeWork_5/ex2_Book/bean/Book.java]: Serializable

**Review by TLittleProgrammer — DISMISSED** [review]
lgfm

**Review by vladis1350 — DISMISSED** [review]
lgfm

**Review by javaGuruBY — APPROVED** [review]
lgfm

**Pull request merged**